src/widget/wmainmenubar.cpp

```diff
@@ -173,20 +173,6 @@ void WMainMenuBar::initialize() {
     QMenu* pViewMenu = new QMenu(tr("&View"));
 
     QString mayNotBeSupported = tr("May not be supported on all skins.");
-    QString showSamplersTitle = tr("Show Samplers");
```

Comment:
Why not introduce here a simple variable which tell if the current selected Skin has a "Show Samplers" Button or not, depending on that, show here this option, that would also solve @daschuer remark 
> Unfortunately this change also breaks various forum and legacy skins

new Skins which have those Buttons sets a variable and then the option is not visible in the menu, otherwise it is visible in the menu.


if(selected_skin.has_sample_button==0)
(
 QString showSamplersTitle = tr("Show Samplers");
 QString showSamplersText = tr("Show the sample deck section of the Mixxx interface.") +
          " " + mayNotBeSupported;
  auto pViewShowSamplers = new QAction(showSamplersTitle, this);
  pViewShowSamplers->setCheckable(true);
 pViewShowSamplers->setShortcut(
     QKeySequence(m_pKbdConfig->getValue(ConfigKey("[KeyboardShortcuts]",
                                               "ViewMenu_ShowSamplers"),
                                                tr("Ctrl+1", "Menubar|View|Show Samplers"))));
   pViewShowSamplers->setStatusTip(showSamplersText);
   pViewShowSamplers->setWhatsThis(buildWhatsThis(showSamplersTitle, showSamplersText));
   createVisibilityControl(pViewShowSamplers, ConfigKey("[Samplers]", "show_samplers"));
   pViewMenu->addAction(pViewShowSamplers);
)

Reply:
Unfortunately it is not that simple because the tangled initialization process makes things fragile. I would rather add toggles to Shade and work towards removing the menu bar entirely.

Reply:
@nopeppermint would you like to add those buttons to Shade?

Reply:
I may try to do this, if will try to add some kind of "extensive menu" as in deere/tango, thats seems to be the best solution for me

Reply:
Shade has this button already.
The tiny text labels above the vu-meters.
They are not very usable though. 

I am currently working on a Shade Skin that solved it. The concept will re-use the current looping region for basic quick access buttons for various features, and toggle buttons to open additional skin regions with fine access to each feature.

Reply:
I am happy to have a helping hand for this. Unfortunately I have no access to my device with my work in progress Shade skin. I will publish it tomorrow.